tasks.py

```diff
@@ -221,29 +222,40 @@ def docker(ctx, name="gx38local", tag="latest", build=False, cmd="bash"):
             "The docker task must be invoked from the same directory as the task.py file at the top of the repo.",
             code=1,
         )
+
+    cmds = ["docker"]
+
     if build:
-        cmds = [
-            "docker",
-            "buildx",
-            "build",
-            "-f",
-            "docker/Dockerfile.tests",
-            f"--tag {name}:{tag}",
-            *[f"--build-arg {arg}" for arg in ["SOURCE=local", "PYTHON_VERSION=3.8"]],
-            ".",
-        ]
+        cmds.extend(
+            [
+                "buildx",
+                "build",
+                "-f",
+                "docker/Dockerfile.tests",
+                f"--tag {name}:{tag}",
+                *[
+                    f"--build-arg {arg}"
+                    for arg in ["SOURCE=local", "PYTHON_VERSION=3.8"]
```

Comment:
We've agreed on 3.8 as the standard here? Or should we test with our minimum supported version 3.7?

Reply:
I can make this another CLI argument.